app/javascript/availability/libraries_locations.json

```diff
@@ -756,15 +765,15 @@
     "UP-SOCSCI": "Social Sciences & Education Library (Paterno)",
     "UP-EDUBEHV": "Social Sciences & Education Library (Paterno)",
     "UP-SPECCOL": "Special Collections Library (Paterno)",
-    "WILKESBAR": "Wilkes-Barre",
+    "WILKESBAR": "Penn State Wilkes-Barre",
     "WITHDRAWN": "Withdrawn",
     "WORLD": "World Campus",
-    "WSCRANTON": "Scranton",
+    "WSCRANTON": "Penn State Scranton",
     "XTERNAL": "Associated Libraries",
-    "YORK": "York"
+    "YORK": "Penn State York"
   },
   "request_via_ill": {
-    "BINDERY-DN": "Copy unavailable, request via Interlibrary Loan",
+    "BINDERY-DN": "Copy unavailable, request via Interlibrary Loan".
```

Comment:
@ruthtillman - looks like you have a period here in the JSON instead of a comma, which is a syntax error. This may be what is causing a bunch of tests to fail. I bet if you swapped this back out CI will be green again 🟢  🙂

Reply:
oh good grief. Thanks so much for the review.